src/main/java/com/netgrif/application/engine/petrinet/web/PetriNetController.java

```diff
@@ -191,7 +195,7 @@ public MessageResource deletePetriNet(@PathVariable("id") String processId, Auth
             return MessageResource.errorMessage("Deleting Petri net " + processId + " failed!");
         }
         LoggedUser user = (LoggedUser) auth.getPrincipal();
-        this.service.deletePetriNet(decodedProcessId, user);
+        asyncRunner.execute(() -> this.service.deletePetriNet(decodedProcessId, user));
         return MessageResource.successMessage("Petri net " + decodedProcessId + " was deleted");
```

Comment:
I would change this message since the deletion process is not over when the response is sent.
for example: `Petri net [ID] is being deleted`

Reply:
Corrected